tools/arch/PKGBUILD

```diff
@@ -42,20 +41,24 @@ build() {
     --disable-silent-rules \
     --with-admin-group=wheel \
     --with-cockpit-user=cockpit-ws \
-    --with-cockpit-ws-instance-user=cockpit-wsinstance
-
+    --with-cockpit-ws-instance-user=cockpit-wsinstance \
+    --enable-pybridge
```

Comment:
This is no more since PR #19246; the opposite `--enable-old-bridge` isn't for arch, so you can just drop this.

Reply:
Ok, will drop it after the triggered test run. And sync upstream :)